app/views/propertyrepresentation/manage/myAgents.scala.html

```diff
@@ -0,0 +1,69 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import actions.requests.BasicAuthenticatedRequest
+@import uk.gov.hmrc.govukfrontend.views.html.components.implicits._
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import models.propertyrepresentation.ManageAgentOptionItem
+@import models.propertyrepresentation.ManageAgentOptions
+@import models.propertyrepresentation.UnassignFromAllProperties
+@import models.propertyrepresentation.ManageAgentRequest
+@import models.propertyrepresentation.AgentList
+@import models.propertyrepresentation.AgentSummary
+@import config.ApplicationConfig
+
+@this(govukTable : GovukTable, mainLayout: views.html.mainLayout)
+
+@(agents: List[AgentSummary], propertyLinkingCount: Int)(implicit request: Request[_], messages: Messages, config: ApplicationConfig)
+
+@mainLayout(
+    pageTitle = messages("propertyRepresentation.showAgents.title"),
+    backUri = Some(config.newDashboardUrl("home"))
```

Comment:
is this the permanent backUrl? just curious if there is any other places this will go to.

Reply:
I am using kind of Temporary pointing to Home page because We are going to get this page from Home page, I need to confirm with Karen and Habeeb by tmw
I think this page will be called from one of Habeeb new page,

Reply:
Karren said , it should be link back to home page